config/sunspot.yml

```diff
@@ -1,24 +1,21 @@
+---
 production:
   solr:
     hostname: localhost
     port: 8983
     log_level: WARNING
-    path: /solr/default
```

Comment:
This stuff is unfortunate -- it keeps getting overwritten with new settings when rake test is run (we edit the YAML file to disable solr during testing; not ideal but we couldn't find another way), and the extra lines here keep getting added back in. Nothing needs to change in this PR but if you do think of a better solution to avoid this, please feel free to suggest. Thanks!